QuickOSM/resources/ui/main_window.ui

```diff
@@ -545,7 +545,14 @@ QListWidget::item::selected {
                <item>
                 <widget class="QPushButton" name="button_show_query">
                  <property name="text">
-                  <string>Show query</string>
+                  <string>Show query in XML</string>
+                 </property>
+                </widget>
+               </item>
+               <item>
+                <widget class="QPushButton" name="button_show_query_oql">
+                 <property name="text">
+                  <string>Show query in OQL</string>
```

Comment:
Maybe make a drop down menu on the button.
Like in QGIS Vector properties dialog, at the bottom, the "Style" button.

The best maybe would be like the selection vector toolbar when you have a button by default even when not clicking on the arrow. You see what I mean ? Look at the ruler button in the QGIS tool bar ( la petite règle) pour changer entre angle, distance, surface). There is always the first button by default.